pkg/services/rpcsrv/server_test.go

```diff
@@ -2617,20 +2891,59 @@ func encodeBlock(t *testing.T, b *block.Block) string {
 	return base64.StdEncoding.EncodeToString(w.Bytes())
 }
 
+func encodeTx(t *testing.T, tx *transaction.Transaction) string {
+	w := io.NewBufBinWriter()
+	tx.EncodeBinary(w.BinWriter)
```

Comment:
You can use `testserdes.EncodeBinary(tx)`.

Reply:
Thanks!